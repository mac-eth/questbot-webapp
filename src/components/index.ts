import Navbar from "./Navbar";
import EarnRewards from "./EarnRewards";
import Gamify from "./Gamify";
import Business from "./Business";
import Clients from "./Clients";
import CTA from "./CTA";
import Stats from "./Stats";
import Footer from "./Footer";
import Testimonials from "./Testimonials";
import Hero from "./Hero";

export {
  Navbar,
  EarnRewards,
  Gamify,
  Business,
  Clients,
  CTA,
  Stats,
  Footer,
  Testimonials,
  Hero,
};
