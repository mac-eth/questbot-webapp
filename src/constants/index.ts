import { people01, people02, people03, facebook, instagram, linkedin, twitter, metamask, binance, coinbase, dropbox, shield, star, game, discord  } from "../assets";

export const navLinks = [
  {
    id: "",
    title: "Home",
  },
  {
    id: "docs",
    title: "Docs",
  },
  {
    id: "dashboard",
    title: "Dashboard",
  },

];

export const ProfileDropdownOptions = [
  { title: "QuestBot",
    Links: [
    { href: "/pro", title: "Pro" },
    { href: "/memberships", title: "Memberships" }
    ]
  },
  { title: "My Servers",
    Links: [
    { href: "/dashboard", title: "My servers" },
    { href: "/", title: "Transfer Premium" },
    ]
  },
  { title: "Payment",
  Links: [
  { href: "/billing", title: "Billing" },
  ]
  },
]

export const features = [
  {
    id: "feature-1",
    icon: star,
    title: "Rewards",
    content:
      "Automated rewards for your members, growing your community and engagement.",
  },
  {
    id: "feature-2",
    icon: shield,
    title: "Built-in Security",
    content:
      "A state of the art Moderation Tool, making sure your community is always safe and secured.",
  },
  {
    id: "feature-3",
    icon: game,
    title: "Gamifying your Community",
    content:
      "Automated games and challenges, making your community more engaging and fun.",
  },
];

export const feedback = [
  {
    id: "feedback-1",
    content:
      "Money is only a tool. It will take you wherever you wish, but it will not replace you as the driver.",
    name: "Herman Jensen",
    title: "Founder & Leader",
    img: people01,
  },
  {
    id: "feedback-2",
    content:
      "Money makes your life easier. If you're lucky to have it, you're lucky.",
    name: "Steve Mark",
    title: "Founder & Leader",
    img: people02,
  },
  {
    id: "feedback-3",
    content:
      "It is usually people in the money business, finance, and international trade that are really rich.",
    name: "Kenn Gallagher",
    title: "Founder & Leader",
    img: people03,
  },
  {
    id: "feedback-1",
    content:
      "Money is only a tool. It will take you wherever you wish, but it will not replace you as the driver.",
    name: "Herman Jensen",
    title: "Founder & Leader",
    img: people01,
  },
  {
    id: "feedback-2",
    content:
      "Money makes your life easier. If you're lucky to have it, you're lucky.",
    name: "Steve Mark",
    title: "Founder & Leader",
    img: people02,
  },
  {
    id: "feedback-3",
    content:
      "It is usually people in the money business, finance, and international trade that are really rich.",
    name: "Kenn Gallagher",
    title: "Founder & Leader",
    img: people03,
  },
];

export const stats = [
  {
    id: "stats-1",
    title: "Active Users",
    value: "2.5M+",
  },
  {
    id: "stats-2",
    title: "Active Servers",
    value: "530+",
  },
  {
    id: "stats-3",
    title: "Gold Distributed",
    value: "$230M+",
  },
];

export const footerLinks = [
  {
    title: "Useful Links",
    links: [
      {
        name: "Dashboard",
        link: "",
      },
      {
        name: "Documentation",
        link: "",
      },
      {
        name: "Guides",
        link: "",
      },
      {
        name: "Terms & Services",
        link: "",
      },
    ],
  },
  {
    title: "Community",
    links: [
      {
        name: "Support",
        link: "",
      },
      {
        name: "Twitter",
        link: "",
      },
      {
        name: "Team Socials",
        link: "",
      },
      {
        name: "Reviews",
        link: "",
      },
    ],
  },
  {
    title: "Partner",
    links: [
      {
        name: "QuestBotEarn",
        link: "",
      },
      {
        name: "Become a Partner",
        link: "",
      },
      {
        name: "Careers",
        link: "",
      },
    ],
  },
];

export const socialMedia = [
  {
    id: "social-media-1",
    icon: instagram,
    link: "https://www.instagram.com/",
  },
  {
    id: "social-media-2",
    icon: facebook,
    link: "https://www.facebook.com/",
  },
  {
    id: "social-media-3",
    icon: twitter,
    link: "https://www.twitter.com/",
  },
  {
    id: "social-media-4",
    icon: linkedin,
    link: "https://www.linkedin.com/",
  },
];

export const clients = [
  {
    id: "client-1",
    logo: discord,
  },
  {
    id: "client-2",
    logo: metamask,
  },
];