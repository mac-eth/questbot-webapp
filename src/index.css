@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap");

@tailwind base;
@tailwind components;
@tailwind utilities;

:root {
  --black-gradient: linear-gradient(
    144.39deg,
    #ffffff -278.56%,
    #6d6d6d -78.47%,
    #11101d 91.61%
  );
  --card-shadow: 0px 20px 100px -10px rgba(66, 71, 91, 0.1);
}

* {
  scroll-behavior: smooth;
}

.text-gradient {
  background: radial-gradient(
    64.18% 64.18% at 71.16% 35.69%,
    #eddefa -43.27%,
    #d0bef5 -21.24%,
    #b29df0 12.19%,
    #b67deb 29.82%,
    #935ce6 51.94%,
    #8e33cf 90.29%
  );
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  background-clip: text;
}

.bg-purple-gradient {
  background: linear-gradient(
    157.81deg,
    #eddefa -43.27%,
    #d0bef5 -21.24%,
    #b29df0 12.19%,
    #b67deb 29.82%,
    #935ce6 51.94%,
    #8e33cf 90.29%
  );
}

.bg-black-gradient {
  background: linear-gradient(
    144.39deg,
    #ffffff -278.56%,
    #6d6d6d -78.47%,
    #11101d 91.61%
  );
}

.bg-black-gradient-2 {
  background: linear-gradient(
    -168.39deg,
    #ffffff -278.56%,
    #6d6d6d -78.47%,
    #11101d 91.61%
  );
}

.bg-gray-gradient {
  background: linear-gradient(
    153.47deg,
    rgba(255, 255, 255, 0) -341.94%,
    #14101d 95.11%
  );
}

.bg-discount-gradient {
  background: linear-gradient(125.17deg, #272727 0%, #11101d 100%);
}

.box-shadow {
  box-shadow: 0px 20px 100px -10px rgba(66, 71, 91, 0.1);
}

.sidebar {
  -webkit-animation: slide-top 0.5s cubic-bezier(0.25, 0.46, 0.45, 0.94) both;
  animation: slide-top 0.5s cubic-bezier(0.25, 0.46, 0.45, 0.94) both;
}

@-webkit-keyframes slide-top {
  0% {
    -webkit-transform: translateY(100px);
    transform: translateY(100px);
  }
  100% {
    -webkit-transform: translateY(0);
    transform: translateY(0);
  }
}

@keyframes slide-top {
  0% {
    -webkit-transform: translateY(100px);
    transform: translateY(100px);
  }
  100% {
    -webkit-transform: translateY(0);
    transform: translateY(0);
  }
}

.feature-card:hover {
  background: var(--black-gradient);
  box-shadow: var(--card-shadow);
}

.feedback-container .feedback-card:last-child {
  margin-right: 0px;
}

.feedback-card {
  background: transparent;
}

.feedback-card:hover {
  background: var(--black-gradient);
}

.blue__gradient {
  background: linear-gradient(180deg, rgba(188, 165, 255, 0) 0%, #214d76 100%);
  filter: blur(123px);
}

.pink__gradient {
  background: linear-gradient(90deg, #f4c4f3 0%, #fc67fa 100%);
  filter: blur(900px);
}

.white__gradient {
  background: rgba(255, 255, 255, 0.6);
  filter: blur(750px);
}

/* Glow Button Thingo */

.glow-on-hover {

  border: none;
  outline: none;
  color: #fff;
  background: #111;
  cursor: pointer;
  position: relative;
  z-index: 0;
  border-radius: 50%;
}

.glow-on-hover:before {
  content: '';
  background: linear-gradient(45deg, #ff00d0, #f200ff, #aa00ff, #8800ff, #1e00ff, #005eff);
  position: absolute;
  top: -2px;
  left:-2px;
  background-size: 400%;
  z-index: -1;
  filter: blur(5px);
  width: calc(100% + 4px);
  height: calc(100% + 4px);
  animation: glowing 30s linear infinite;
  opacity: 0;
  transition: opacity .3s ease-in-out;
  border-radius: 50%;
  opacity: 1;
}

.glow-on-hover:active {
  color: #000
}

.glow-on-hover:hover:before {
  filter: blur(30px);
  transition: all .2s ease-in-out;
}

.glow-on-hover:after {
  z-index: -1;
  content: '';
  position: absolute;
  width: 100%;
  height: 100%;
  background: #111;
  left: 0;
  top: 0;
  border-radius: 50%;
}

/* Fade-In Animation */

.fade-in-section {
  opacity: 0;
  transform: translateY(20vh);
  visibility: hidden;
  transition: opacity 1200ms ease-out, transform 600ms ease-out, visibility 1200ms ease-out;
  will-change: opacity, transform, visibility;
}
.fade-in-section.is-visible {
  opacity: 1;
  transform: none;
  visibility: visible;
}

@keyframes glowing {
  0% { background-position: 0 0; }
  50% { background-position: 400% 0; }
  100% { background-position: 0 0; }
}
