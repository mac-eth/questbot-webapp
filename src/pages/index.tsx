import React from "react";
import styles from "../styles/style";
import { EarnRewards, Business, Gamify, Clients, CTA, Footer, Navbar, Stats,  Hero } from "../components/index";

const Home = () => (
  <div className="bg-primary w-full">

    <div className={`bg-primary ${styles.flexStart}`}>
      <div className={`${styles.boxWidth}`}>
        <Navbar />
        <Hero />
        <Stats />
      </div>
    </div>
    
    <div className={`bg-primary ${styles.paddingX} ${styles.flexCenter}`}>
      <div className={`${styles.boxWidth}`}>
        <Business />
        <EarnRewards />
        <Gamify />
        <Clients />
        <CTA />
        <Footer />
      </div>
    </div>
  </div>
);

export default Home;
