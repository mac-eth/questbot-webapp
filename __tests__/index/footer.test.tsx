import { render, screen } from "@testing-library/react";
import Renderer from 'react-test-renderer';
import Footer from "../../src/components/Footer";
import "@testing-library/jest-dom";
import { footerLinks } from "../../src/constants";

describe("Footer Component", () => {
  it("Renders the Footer Logo", () => {
    render(<Footer />);
    const logo = screen.getByRole("img", {
      name: "questbot_logo",
    });
    expect(logo).toBeInTheDocument();
  });
  it("Renders the Footer Description", () => {
    render(<Footer />);
    const text = screen.getByRole("text", {
      name: "description",
    });
    expect(text).toBeInTheDocument();
  });
  it("Renders the Footer Copyright", () => {
    render(<Footer />);
    const text = screen.getByRole("text", {
      name: "copyright",
    });
    expect(text).toBeInTheDocument();
  });
  it("Renders the Footer Link Titles", () => {
    render(<Footer />);
    for (var i = 0; i < footerLinks.length; i++) {
      const text = screen.getByRole("heading", {
        name: `${footerLinks[i].title}`,
      });
      expect(text).toBeInTheDocument();
    }
  });
  it("Renders the Footer Links", () => {
    render(<Footer />);
    for (var i = 0; i < footerLinks.length; i++) {
      for (var j = 0; j < footerLinks[i].links.length; j++) {
        const text = screen.getByRole("listitem", {
          name: `${footerLinks[i].links[j].name}`,
        });
        expect(text).toBeInTheDocument();
      }
    }
  });
  it("Renders the entire Footerw", () => {
    const renderer = Renderer.create(
      <Footer />
    );
    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
