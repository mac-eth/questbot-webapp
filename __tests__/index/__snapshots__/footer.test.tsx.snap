// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Renders the entire Footerw 1`] = `
<div
  style={
    {
      "transform": "translateY(100px) translateZ(0)",
    }
  }
>
  <section
    className="flex justify-center items-center sm:py-16 py-6 flex-col"
  >
    <div
      className="flex justify-center items-start md:flex-row flex-col mb-8 w-full"
    >
      <div
        className="flex-[1] flex flex-col justify-start mr-10"
      >
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2740%27%20height=%2740%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="questbot_logo"
            className="w-[266px] h-[72.14px] object-contain"
            data-nimg="intrinsic"
            decoding="async"
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="questbot_logo"
              className="w-[266px] h-[72.14px] object-contain"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
              srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
        <div
          aria-label="description"
          className="font-poppins font-normal text-dimWhite text-[18px] leading-[30.8px] mt-4 max-w-[312px]"
          role="text"
        >
          Boosting engagement with the next generation of Play2Earn.
        </div>
      </div>
      <div
        className="flex-[1.5] w-full flex flex-row justify-between flex-wrap md:mt-0 mt-10"
      >
        <div
          aria-label="Useful Links"
          className="flex flex-col ss:my-0 my-4 min-w-[150px]"
        >
          <h4
            className="font-poppins font-medium text-[18px] leading-[27px] text-white"
          >
            Useful Links
          </h4>
          <ul
            className="list-none mt-4"
          >
            <li
              aria-label="Dashboard"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-4"
            >
              Dashboard
            </li>
            <li
              aria-label="Documentation"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-4"
            >
              Documentation
            </li>
            <li
              aria-label="Guides"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-4"
            >
              Guides
            </li>
            <li
              aria-label="Terms & Services"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-0"
            >
              Terms & Services
            </li>
          </ul>
        </div>
        <div
          aria-label="Community"
          className="flex flex-col ss:my-0 my-4 min-w-[150px]"
        >
          <h4
            className="font-poppins font-medium text-[18px] leading-[27px] text-white"
          >
            Community
          </h4>
          <ul
            className="list-none mt-4"
          >
            <li
              aria-label="Support"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-4"
            >
              Support
            </li>
            <li
              aria-label="Twitter"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-4"
            >
              Twitter
            </li>
            <li
              aria-label="Team Socials"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-4"
            >
              Team Socials
            </li>
            <li
              aria-label="Reviews"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-0"
            >
              Reviews
            </li>
          </ul>
        </div>
        <div
          aria-label="Partner"
          className="flex flex-col ss:my-0 my-4 min-w-[150px]"
        >
          <h4
            className="font-poppins font-medium text-[18px] leading-[27px] text-white"
          >
            Partner
          </h4>
          <ul
            className="list-none mt-4"
          >
            <li
              aria-label="QuestBotEarn"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-4"
            >
              QuestBotEarn
            </li>
            <li
              aria-label="Become a Partner"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-4"
            >
              Become a Partner
            </li>
            <li
              aria-label="Careers"
              className="font-poppins font-normal text-[16px] leading-[24px] text-dimWhite hover:text-secondary cursor-pointer mb-0"
            >
              Careers
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="w-full flex justify-between items-center md:flex-row flex-col pt-6 border-t-[1px] border-t-[#3F3E45]"
    >
      <div
        aria-label="copyright"
        className="font-poppins font-normal text-center text-[18px] leading-[27px] text-white"
        role="text"
      >
        Copyright â’¸ 2022 QuestBot. All Rights Reserved.
      </div>
      <div
        className="flex flex-row md:mt-0 mt-6"
      >
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2740%27%20height=%2740%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="social-media-1"
            className="w-[21px] h-[21px] object-contain cursor-pointer mr-6"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="social-media-1"
              className="w-[21px] h-[21px] object-contain cursor-pointer mr-6"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
              srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2740%27%20height=%2740%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="social-media-2"
            className="w-[21px] h-[21px] object-contain cursor-pointer mr-6"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="social-media-2"
              className="w-[21px] h-[21px] object-contain cursor-pointer mr-6"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
              srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2740%27%20height=%2740%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="social-media-3"
            className="w-[21px] h-[21px] object-contain cursor-pointer mr-6"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="social-media-3"
              className="w-[21px] h-[21px] object-contain cursor-pointer mr-6"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
              srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
        <span
          style={
            {
              "background": "none",
              "border": 0,
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "width": "initial",
            }
          }
        >
          <span
            style={
              {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "padding": 0,
                "width": "initial",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2740%27%20height=%2740%27/%3e"
              style={
                {
                  "background": "none",
                  "border": 0,
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            />
          </span>
          <img
            alt="social-media-4"
            className="w-[21px] h-[21px] object-contain cursor-pointer mr-0"
            data-nimg="intrinsic"
            decoding="async"
            onClick={[Function]}
            onError={[Function]}
            onLoad={[Function]}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="social-media-4"
              className="w-[21px] h-[21px] object-contain cursor-pointer mr-0"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              onClick={[Function]}
              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
              srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
              style={
                {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
    </div>
  </section>
</div>
`;
